<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAssetWindow</class>
 <widget class="QDialog" name="AddAssetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Asset</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>707</width>
         <height>435</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Asset Manager</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AssetIdLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Asset ID: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AssetIdData"/>
        </item>
        <item>
         <widget class="QLabel" name="CategoryIdLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Category ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="CategoryIdData"/>
        </item>
        <item>
         <widget class="QLabel" name="NameLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NameData"/>
        </item>
        <item>
         <widget class="QLabel" name="TagLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Tag:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TagData"/>
        </item>
        <item>
         <widget class="QLabel" name="DescriptionLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DescriptionData"/>
        </item>
        <item>
         <widget class="QLabel" name="LocationLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LocationData"/>
        </item>
        <item>
         <widget class="QLabel" name="OriginalCostLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Original Cost:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="OriginalCostData"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>20</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
